beaker-vue/src/components/notebook/BeakerNotebookPanel.vue

```diff
@@ -157,6 +157,8 @@ defineExpose({
     width: 100%;
     height: 100%;
 
+    z-index: 50;
```

Comment:
This is key: enabled z-index hierarchy that allows the contents of the NotebookPanel to render over the SideMenu z-index properties (hence lint popup hovers over the other UI elements correctly).